kafka-connect-s3/src/test/java/io/confluent/connect/s3/S3SinkConnectorConfigTest.java

```diff
@@ -102,6 +103,15 @@ public void testRecommendedValues() throws Exception {
   }
 
   @Test
+  public void testAvroDataConfigSupported() throws Exception {
+    properties.put(AvroDataConfig.ENHANCED_AVRO_SCHEMA_SUPPORT_CONFIG, "true");
```

Comment:
This test could actually test the `avroDataConfig` method mentioned above, and make sure the override of schema cache size takes place too.